src/test/scala-2.12/org/apache/james/gatling/simulation/SimulationOnMailCorpus.scala

```diff
@@ -27,7 +27,7 @@ trait SimulationOnMailCorpus {
 
   protected val feeder: SourceFeederBuilder[String] = UserFeeder.toFeeder(getUsers).circular
 
-  protected def injectUsersInScenario(scenario: ScenarioBuilder, nbUsers: UsersDensity = UsersPerHour(50000)) = {
+  protected def injectUsersInScenario(scenario: ScenarioBuilder, nbUsers: UsersDensity = UsersPerSecond(50000)) = {
```

Comment:
50.000 user per second? 

I understand it is a default and that simulation specify their own value but maybe that should be lowered....

Reply:
No I just tried to rename it and obviously the renaming is wrong :)

What it does it takes the number of users / duration in seconds... I put back the duration at 1 hour so it goes to around 13 users per second 

I'm not sure how should I name this class... UsersPerHour is wrong (UsersPerSecond is not better)